nimble/host/include/host/ble_hs_log.h

```diff
@@ -39,6 +33,13 @@
  * @{
  */
 
+#include <nimble/nimble_npl_log.h>
+
+#ifndef BLE_NPL_LOG_MODULE
+/** Defines the logging module for NimBLE Porting Layer (NPL). */
+#define BLE_NPL_LOG_MODULE BLE_HS_LOG
```

Comment:
ahh, forgot about this

this needs to be defined before #include <nimble/nimble_npl_log.h>